spec/requests/api/v1/user_request_spec.rb

```diff
@@ -112,4 +112,32 @@
     expect(User.count).to eq(0)
     expect{User.find(user.id)}.to raise_error(ActiveRecord::RecordNotFound)
   end
+
+  it "can Return an array of users with same location and pace" do
+    user1 = create(:user, username: "jane", location: "denver", estimated_mile_pace: "08:00")
```

Comment:
First three users have the same location as the logged in user and should be matched whereas the last two users will not match.